--- a/src/VBox/HostDrivers/Support/SUPDrv.cpp
+++ b/src/VBox/HostDrivers/Support/SUPDrv.cpp
@@ -5762,7 +5762,8 @@
         /*
          * Done!
          */
-        SUPR0Printf("vboxdrv: %RKv %s\n", pImage->pvImage, pImage->szName);
+        SUPR0Printf("vboxdrv: %#RX64 %s\n",
+                    (uint64_t)(uintptr_t)pImage->pvImage, pImage->szName);
         pReq->u.Out.uErrorMagic = 0;
         pReq->u.Out.szError[0]  = '\0';
     }
